charts/velero/OWNERS

```diff
@@ -1,10 +1,12 @@
 approvers:
-- hectorj2f
+- skriss
 - nrb
 - carlisia
 - ashish-amarnath
+- cpanato
 reviewers:
-- hectorj2f
+- skriss
 - nrb
 - carlisia
 - ashish-amarnath
+- cpanato
```

Comment:
@carlisia my email is `ctadeu[at]gmail.com`

can we make this list in alphabetical order or there is a reason to be like this?

Reply:
Good point! And I'll add you.